### PR TITLE
melee range increased

### PR DESCRIPTION
**Pull request by kattalainen**
Increased base melee range (and the melee radius upgrade scaling) and moved the hitbox forwards a tiny bit. Screenshot of levels and sizes: https://drive.google.com/file/d/1nW-453elhavJXufAVmdJ0-puX8O_SnTl/view?usp=sharing

### CODE REVIEW

**Review by fredrix96 — DISMISSED** [review]
Feels much better

File: StortSpelProjekt/Game/src/Gamefiles/Components/MeleeComponent.cpp [hunk 1]
```diff
@@ -93,7 +90,7 @@ void component::MeleeComponent::Update(double dt)
 
 	float3 position = parentTransform->GetPositionFloat3();
 	float3 forwardVector = parentTransform->GetForwardFloat3();
-	float distanceFromPlayer = m_HalfSize.z + (modelDim.z * parentTransform->GetScale().z / 2.0f);
+	float distanceFromPlayer = m_HalfSize.z + (modelDim.z * parentTransform->GetScale().z);
```
Comment (LeoWikstrom) [inline StortSpelProjekt/Game/src/Gamefiles/Components/MeleeComponent.cpp]: This makes the bounding box appear at different distances from the player depending on the level of the upgrade. The following code should be better:
`float distanceFromPlayer = (m_HalfSize.z * m_ZScale) + (modelDim.z * parentTransform->GetScale().z / 2.0f);`

**Review by LeoWikstrom — CHANGES_REQUESTED** [review]
Much better, though I have one remark

**Review by LeoWikstrom — APPROVED** [review]
Nice 👍🏽

**Review by fredrix96 — APPROVED** [review]

**Pull request merged**